src/test/scala/io/iohk/ethereum/ledger/InMemoryWorldStateProxySpec.scala

```diff
@@ -9,6 +9,12 @@ import org.spongycastle.util.encoders.Hex
 
 class InMemoryWorldStateProxySpec extends FlatSpec with Matchers {
 
+  def persistIfHashMatches(stateRootHash: ByteString, worldState: InMemoryWorldStateProxy): InMemoryWorldStateProxy = {
```

Comment:
@ntallar I would remove this and just call persist in tests. Actually, if you just persist the test works too.